teams/wg-cli.toml

```diff
@@ -2,21 +2,23 @@ name = "wg-cli"
 wg = true
 
 [people]
-leads = []
+leads = ["spacekookie"]
 members = [
     "codesections",
     "Dylan-DPC",
     "epage",
     "killercup",
     "matthiasbeyer",
-    "spacekookie",
     "XAMPPRocky",
     "yoshuawuyts",
 ]
 
 [website]
 page = "cli"
-name = "Command-line interfaces (CLI) working group"
-description = "Focusing on the end-to-end experience of writing CLI apps, both large and small, in Rust."
+name = "Terminal applications  working group"
+description = "Focusing on the end-to-end experience of writing terminal apps, both large and small, in Rust."
 discord-invite = "https://discord.gg/BXCBYX6"
 discord-name = "#wg-cli"
+
+[[lists]]
+address = "embedded-wg@rust-lang.org"
```

Comment:
I guess this is a copy-paste artifact here, is it? :smile:

Reply:
Yes :see_no_evil: